test/cli_update_test.go

```diff
@@ -115,6 +115,32 @@ func (suite *PouchUpdateSuite) TestUpdateCpuPeriod(c *check.C) {
 	c.Assert(metaJSON[0].HostConfig.CPUPeriod, check.Equals, int64(2000))
 }
 
+// TestUpdateCpuMemoryFail is to verify the invalid value of updating container cpu and memory related flags will fail.
+func (suite *PouchUpdateSuite) TestUpdateCpuMemoryFail(c *check.C) {
+	name := "update-container-cpu-memory-period-fail"
+
+	res := command.PouchRun("run", "-d", "--name", name, busyboxImage, "top")
+	defer DelContainerForceMultyTime(c, name)
+	res.Assert(c, icmd.Success)
+
+	output := command.PouchRun("inspect", name).Stdout()
+	result := []types.ContainerJSON{}
```

Comment:
I don't think we need `result` here.  The `inspect` action should be tested by other suite.

Reply:
Yes, it can be removed.